src/App/Hangouts/Commands.elm

```diff
@@ -1,34 +1,45 @@
 module App.Hangouts.Commands exposing (..)
 
 import Http as H
-import HttpBuilder exposing (withExpect)
-import Json.Decode as Decode exposing (Decoder, succeed, string, int)
+import HttpBuilder exposing (withExpect, withHeader, withJsonBody, send)
```

Comment:
Since we're qualifying it's usage with `HttpBuilder`, we need not expose it separately.